internal/pkg/cli/deploy/deploy.go

```diff
@@ -1210,8 +1214,13 @@ func validateAppVersionForAlias(appName string, appVersionGetter versionGetter)
 }
 
 func (d *backendSvcDeployer) validateALBRuntime() error {
-	if d.backendMft.RoutingRule.EmptyOrDisabled() || d.backendMft.RoutingRule.Alias.IsEmpty() || !d.env.HasImportedCerts() {
+	switch {
+	case d.backendMft.RoutingRule.EmptyOrDisabled():
 		return nil
+	case !d.backendMft.RoutingRule.Alias.IsEmpty() && !d.env.HasImportedCerts():
```

Comment:
what if the env has imported certs but alias is not set. Are we good with that?

Reply:
i think so? it would make sense to me to just use `http` in that case

Reply:
But then what domain do we use for them 🤔

Reply:
umm... just the default lb alias? so just a listener rule without any `host-header` restriction?

though i'm realizing that could cause a weird situation where an A-record points to the LB and then this listener rule picks it up 🤔